tests/python-pytest/onnx/test_operators.py

```diff
@@ -378,3 +378,27 @@ def test_onnx_export_contrib_BilinearResize2D(tmp_path, dtype, params):
     x = mx.nd.arange(0, 160).reshape((2, 2, 5, 8))
     M = def_model('contrib.BilinearResize2D', **params)
     op_export_test('contrib_BilinearResize2D', M, [x], tmp_path)
+
+
+@pytest.mark.parametrize("dtype", ["float32", "float64", "int32", "int64"])
```

Comment:
shall we also try float16

Reply:
Update to work with float16 and also test it :) Thanks for the help.